src/components/keymap/add/keymap-add.component.ts

```diff
@@ -1,28 +1,36 @@
 import { Component } from '@angular/core';
+import { Store } from '@ngrx/store';
+
+import 'rxjs/add/operator/count';
+import 'rxjs/add/operator/debounceTime';
+import { Observable } from 'rxjs/Rx';
```

Comment:
Import from `rxjs/Observable`. We will never want to import from `rxjs/Rx` because it includes the whole library.